### DISCUSSION TITLE
Inputs param causes parsing errors

### DISCUSSION THREAD

**Discussion by kvaghela** [convictional/trigger-workflow-and-wait#30] [Q&A] [answered]
Hi, 
I've also failed to get the inputs to work. It can only use the default {}, any data within throws parsing errors.
Tried inputs: {\"name\": \"test\"} and inputs: {"name": "test"}
parse error: Invalid numeric literal at line 1, column 3

It's a real shame as it's a good function. @keithconvictional  Any thoughts on why the inputs don't work?

Thanks,
Kunaal

**Comment by kvaghela** [comment, marked as answer]
Got this working now. On your downstream repo setup workflow_dispatch to take inputs e.g.

on:
  workflow_dispatch:
    # Inputs the workflow accepts.
    inputs:
      testurl:
        description: 'Which URL to test with'
        default: 'http://abcd.com'
        required: true


From your invoker repo yaml using trigger-workflow-and-wait, add this tag

inputs: '{ "testurl" :"http://xyz.com" }'

**Reply by AshwinNmbr** [reply]
do we add it like this? 

 needs: [PublishPackages, SetVersion]
    steps:
      - name: Trigger package update in VSDavinci
        uses: convictional/trigger-workflow-and-wait@v1.6.1
        with:
         inputs: '{ "testurl" :"http://xyz.com" }'
         owner: ${{ github.repository_owner }}
